readit/cli.py

```diff
@@ -49,17 +51,31 @@ def main(insert, add, tag, delete, clear,
                 validate_url = requests.get(url)
                 validate_code = validate_url.status_code
                 if validate_code == 200:
-                    database_connection.add_url(url)
+                    return_value = database_connection.add_url(url)
```

Comment:
How about using using `is_url_added` instead of `return_value`?

Reply:
Yes, It will be more readable.
Thanks